cmd/traefik/traefik.go

```diff
@@ -248,3 +254,41 @@ func checkNewVersion() {
 		}
 	})
 }
+
+func stats(globalConfiguration *configuration.GlobalConfiguration) {
+	if globalConfiguration.SendAnonymousUsage {
+		log.Info(`
+Stats collection is enabled.
+Many thanks to contribute to Traefik improvement by allowing us to receive anonymous information from your configuration.
+Help us improve Traefik by leaving this feature on :)
+More details on: https://docs.traefik.io/basic/#collected-data
+`)
+		collect(globalConfiguration)
+	} else {
+		log.Info(`
+Stats collection is disabled.
+Help us improve Traefik by turning this feature on :)
+More details on: https://docs.traefik.io/basic/#collected-data
+`)
+	}
+}
+
+func collect(globalConfiguration *configuration.GlobalConfiguration) {
+	ticker := time.NewTicker(24 * time.Hour)
+	safe.Go(func() {
+		time.Sleep(10 * time.Minute)
```

Comment:
I'm not sure understand why this first call of `Collect` is required 👼 I guess you want the first one after 10min and then each and every day, right.

Reply:
exactly, we wait 10 min for eliminate some quick restart.

Reply:
Can we document the delay in `basics.md` as well?